guide/source/writing-atmosphere-packages.md

```diff
@@ -294,7 +294,7 @@ You can read more about testing in Meteor in the [Testing article](testing.html)
 
 <h2 id="publishing-atmosphere">Publishing your package</h2>
 
-To publish your package to Atmosphere, run [`meteor publish`](http://docs.meteor.com/#/full/meteorpublish) from the package directory. To publish a package the package name must follow the format of `username:my-package` and the package must contain a [SemVer version number](#version-constraints).
+To publish your package to Atmosphere for the first time, run [`meteor publish --create`](http://docs.meteor.com/#/full/meteorpublish) from the package directory. The package name must follow the format of `username:my-package` and the package must contain a [SemVer version number](#version-constraints). When you want to publish an update to your package, change the version number in `package.js` and then run `meteor publish`.
```

Comment:
It should also mention that after `meteor publish --create`, you should just use `meteor publish`

Reply:
It does :) 

> When you want to publish an update to your package, change the version number in `package.js` and then run `meteor publish`